components/index/VolunteerHero.vue

```diff
@@ -0,0 +1,113 @@
+<template>
```

Comment:
Components are broken up by page. So index is for the main page. You should create a `volunteer` folder alongside `index` and then just name the components `Hero`, `How`, etc.

We might look at breaking out some components to being common, like the testimonials.